src/modules/krita/producer_krita.yml

```diff
@@ -0,0 +1,33 @@
+schema_version: 0.3
+type: producer
+identifier: krita
+title: Krita
+version: 1
+copyright: Eoin O'Neill, Emmet O'Neill
+creator: Eoin O'Neill, Emmet O'Neill
+license: LGPLv2.1
+language: en
+tags:
+  - Video
+  - Audio
+parameters:
```

Comment:
The `resource` property is not documented. It gets a `argument: yes` attribute. Also, the `speed` property is not documented.